helm-chart/renku-notebooks/templates/statefulset.yaml

```diff
@@ -87,9 +87,9 @@ spec:
             - name: NB_SSH_ENABLED
               value: {{ .Values.ssh.enabled | quote }}
             - name: NB_SESSIONS__CULLING__REGISTERED__IDLE_SECONDS
-              value: {{ .Values.culling.idleThresholdSeconds.registered | quote }}
+              value: {{ .Values.culling.idleSecondsThreshold.registered | quote }}
             - name: NB_SESSIONS__CULLING__ANONYMOUS__IDLE_SECONDS
-              value: {{ .Values.culling.idleThresholdSeconds.anonymous | quote }}
+              value: {{ .Values.culling.idleSecondsThreshold.anonymous | quote }}
```

Comment:
You are missing here somewhere the hibernation threshold, right? Same in the test.yaml.